test/e2e/installplan_e2e_test.go

```diff
@@ -2574,7 +2574,6 @@ func TestInstallPlanCRDValidation(t *testing.T) {
 }
 
 func TestInstallPlanUnpacksBundleImage(t *testing.T) {
-	defer cleaner.NotifyTestComplete(t, true)
```

Comment:
Is this intentional?

Reply:
yes -- it was attempting to deleted resources that no longer existed, and I didn't want to dig into the cleanup stuff for this PR. It's not needed here anyway since the namespace cleanup is explicitly deferred below.